includes/betweenUpdates.php

```diff
@@ -366,6 +366,10 @@ function ensureTableAndColumns($db, $tableName, $expectedColumns, $renameColumns
 if (!$r=db_fetch_array(db_select($qtxt,__FILE__ . " linje " . __LINE__))) {
 	db_modify("ALTER table regnskab ADD column sms integer",__FILE__ . " linje " . __LINE__);
 }
+$qtxt="SELECT column_name FROM information_schema.columns WHERE table_name='online' and column_name='session_id'";
+if (!$r=db_fetch_array(db_select($qtxt,__FILE__ . " linje " . __LINE__))) {
```

Comment:
Note that this checks if the column does not exsist, something like
```
SELECT column_name, data_type FROM information_schema.columns WHERE table_name = 'online' AND column_name = 'session_id'
```
Would also fetch the datatype

Reply:
Better check now in `login.php`